src/main/java/bio/terra/cli/businessobject/Workspace.java

```diff
@@ -312,6 +325,18 @@ public String grantBreakGlass(
     return granteeProxyGroupEmail;
   }
 
+  public static Properties stringMapToProperties(Map<String, String> map) {
+    Properties propertyResult = new Properties();
+    if (map == null) {
```

Comment:
Can delete 330-332

Reply:
Mark the map and the function `@Nullable` if you keep these lines.